### PR TITLE
doc(tutorials): duplicate tutorial structure

### PR DESCRIPTION
**Pull request by stephanbcbauer**
## Description

This PR duplicates the current E2E Adopter Journey tutorial because we need a basis to work on the new tutorial. The old one still works. The old one will be deleted after the new one is ready.

image

image

image

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by stephanbcbauer** [comment]
> Why do we need a work in progress documentation on our live website and marking the previous one as deprecated? Why not working on the branch `doc/add-new-tutorial` and use Docusaurus Dev server (`npm start`) for rendering checks until it's finished and publish then the new version of the tutorial?

We have to make plenty of changes to the future tutorial (for the Tractus-X community days next week). This is currently only possible step by step because otherwise the pull request would be far too large, and we would have to have the changes approved by Eclipse.  Currently, 5 people are working on the tutorial at the same time. Working on a “base” is therefore expedient. Without influencing the old tutorial.

File: docs/tutorials/e2e_deprecated/connect/assets/img.png
Comment (SebastianBezold) [inline docs/tutorials/e2e_deprecated/connect/assets/img.png:1]: The filename is a bit general

File: docs/tutorials/e2e_deprecated/boost/assets/components_contract_definition.png
Comment (SebastianBezold) [inline docs/tutorials/e2e_deprecated/boost/assets/components_contract_definition.png:1]: Would it be possible to add this as `.svg`. Would allow for easier changes

**Review by SebastianBezold — CHANGES_REQUESTED** [review]
Since the tutorial will not be versioned, I would move all assets to the `/static` directory. 
Maybe to a `/static/tutorial` sub-directory to have it all grouped

**Review by JohannesARENA2036 — APPROVED** [review]
We should approve this branch, even if we still need to do a couple of  further updates
Thanks Johannes

**Comment by SebastianBezold** [comment]
> We should approve this branch, even if we still need to do a couple of further updates Thanks Johannes

That is exactly what we should NOT do. PRs should include a finished peace, that provides value instead of just throwing junks on this repo, that are potentially finished later. I totally agree with what @carslen said. 
I also don't really understand why a tutorial is "deprecated". 
Maybe the website is just the wrong place to put this content

File: docs/tutorials/e2e_deprected.md
Comment (SebastianBezold) [inline docs/tutorials/e2e_deprected.md:65]: If this should stay at least for a bit, then at least the links have to be adapted. 
I would still prefer, that this PR does delete the old and add the new on one go.

**Comment by carslen** [comment]
> We should approve this branch, even if we still need to do a couple of further updates Thanks Johannes

Why should we? I would have expected arguments rather than an additional request for approval.

**Comment by carslen** [comment]
> > Why do we need a work in progress documentation on our live website and marking the previous one as deprecated? Why not working on the branch `doc/add-new-tutorial` and use Docusaurus Dev server (`npm start`) for rendering checks until it's finished and publish then the new version of the tutorial?
> 
> We have to make plenty of changes to the future tutorial (for the Tractus-X community days next week). This is currently only possible step by step because otherwise the pull request would be far too large, and we would have to have the changes approved by Eclipse. Currently, 5 people are working on the tutorial at the same time. Working on a “base” is therefore expedient. Without influencing the old tutorial.

I would not count the content of a tutorial as source code. Thus, no IP Check should be necessary even if there are more than 1000 lines of changed. BTW: This PR is already > 1.000 lines of code ;)

I don't get the "5 people working on this tutorial" argument. Why can't they work on the branch this PR is opened from?

**Comment by stephanbcbauer** [comment]
Thank you very much for your comments. I will delete this PR. We will create a “development” branch and merge all the changes, when we are ready.

**Pull request closed without merging**
